products/devstudio/_common/product.yml

```diff
@@ -7,9 +7,9 @@ youtube_album: https://www.youtube.com/playlist?list=PLf3vm0UK6HKrxJ9OtX2Zmgw3ny
 buzz_tags:
   - jbds
 guides:
-  Release_Notes:
-  Install_Red_Hat_JBoss_Developer_Studio:
-  Update_Red_Hat_JBoss_Developer_Studio:
+  Release_Notes: https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_Developer_Studio/9.1-Beta1/html/9.1_Beta1_Release_Notes/index.html
```

Comment:
OK, I've updated the PR. We can't use current_version yet since Beta2 is a pre-release to 9.1.0.GA. So, this PR will update the links to use Beta1 docs, and then later today/tomorrow when the JBDS 9.1.0.Beta2 bits are staged in CSP, I'll send a new PR to move these links to Beta2. Just wanted to ensure these work correctly beforehand.